
















































info [Sun Oct 14 2018 15:10:20 GMT+1100 (AEDT)]  #-- Socket listening on http://localhost port: 8011







































info [Sun Oct 14 2018 15:27:51 GMT+1100 (AEDT)]  ::[mongoUtils]=> connectToDB() => connecting to db with URL: mongodb://userAdmin:baboon_User_Admin@localhost:27018/king_louie
info [Sun Oct 14 2018 15:27:51 GMT+1100 (AEDT)]  { MongoError: Authentication failed.
    at /Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:581:63
    at authenticateStragglers (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:504:16)
    at Connection.messageHandler (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:540:5)
    at emitMessageHandler (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/connection.js:310:10)
    at Socket.<anonymous> (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/connection.js:453:17)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
    at Socket.Readable.push (_stream_readable.js:208:10)
  ok: 0,
  errmsg: 'Authentication failed.',
  code: 18,
  codeName: 'AuthenticationFailed',
  name: 'MongoError',
  [Symbol(mongoErrorContextSymbol)]: {} }




info [Sun Oct 14 2018 15:27:55 GMT+1100 (AEDT)]  ::[mongoUtils]=> connectToDB() => connecting to db with URL: mongodb://userAdmin:baboon_User_Admin@localhost:27018/king_louie
info [Sun Oct 14 2018 15:27:55 GMT+1100 (AEDT)]  { MongoError: Authentication failed.
    at /Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:581:63
    at authenticateStragglers (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:504:16)
    at Connection.messageHandler (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:540:5)
    at emitMessageHandler (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/connection.js:310:10)
    at Socket.<anonymous> (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/connection.js:453:17)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
    at Socket.Readable.push (_stream_readable.js:208:10)
  ok: 0,
  errmsg: 'Authentication failed.',
  code: 18,
  codeName: 'AuthenticationFailed',
  name: 'MongoError',
  [Symbol(mongoErrorContextSymbol)]: {} }



info [Sun Oct 14 2018 15:28:19 GMT+1100 (AEDT)]  ::[mongoUtils]=> connectToDB() => connecting to db with URL: mongodb://userAdmin:baboon_User_Admin@localhost:27018/king_louie
info [Sun Oct 14 2018 15:28:19 GMT+1100 (AEDT)]  { MongoError: Authentication failed.
    at /Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:581:63
    at authenticateStragglers (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:504:16)
    at Connection.messageHandler (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:540:5)
    at emitMessageHandler (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/connection.js:310:10)
    at Socket.<anonymous> (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/connection.js:453:17)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
    at Socket.Readable.push (_stream_readable.js:208:10)
  ok: 0,
  errmsg: 'Authentication failed.',
  code: 18,
  codeName: 'AuthenticationFailed',
  name: 'MongoError',
  [Symbol(mongoErrorContextSymbol)]: {} }



info [Sun Oct 14 2018 15:28:34 GMT+1100 (AEDT)]  ::[mongoUtils]=> connectToDB() => connecting to db with URL: mongodb://userAdmin:baboon_User_Admin@localhost:27018/king_louie
info [Sun Oct 14 2018 15:28:34 GMT+1100 (AEDT)]  { MongoError: Authentication failed.
    at /Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:581:63
    at authenticateStragglers (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:504:16)
    at Connection.messageHandler (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:540:5)
    at emitMessageHandler (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/connection.js:310:10)
    at Socket.<anonymous> (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/connection.js:453:17)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
    at Socket.Readable.push (_stream_readable.js:208:10)
  ok: 0,
  errmsg: 'Authentication failed.',
  code: 18,
  codeName: 'AuthenticationFailed',
  name: 'MongoError',
  [Symbol(mongoErrorContextSymbol)]: {} }



info [Sun Oct 14 2018 15:31:42 GMT+1100 (AEDT)]  ::[mongoUtils]=> connectToDB() => connecting to db with URL: mongodb://userAdmin:baboon_User_Admin@localhost:27018/king_louie
info [Sun Oct 14 2018 15:31:42 GMT+1100 (AEDT)]  { MongoError: Authentication failed.
    at /Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:581:63
    at authenticateStragglers (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:504:16)
    at Connection.messageHandler (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:540:5)
    at emitMessageHandler (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/connection.js:310:10)
    at Socket.<anonymous> (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/connection.js:453:17)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
    at Socket.Readable.push (_stream_readable.js:208:10)
  ok: 0,
  errmsg: 'Authentication failed.',
  code: 18,
  codeName: 'AuthenticationFailed',
  name: 'MongoError',
  [Symbol(mongoErrorContextSymbol)]: {} }



info [Sun Oct 14 2018 15:32:15 GMT+1100 (AEDT)]  ::[mongoUtils]=> connectToDB() => connecting to db with URL: mongodb://userAdmin:baboon_User_Admin@localhost:27018/king_louie
info [Sun Oct 14 2018 15:32:16 GMT+1100 (AEDT)]  { MongoError: Authentication failed.
    at /Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:581:63
    at authenticateStragglers (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:504:16)
    at Connection.messageHandler (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:540:5)
    at emitMessageHandler (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/connection.js:310:10)
    at Socket.<anonymous> (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/connection.js:453:17)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
    at Socket.Readable.push (_stream_readable.js:208:10)
  ok: 0,
  errmsg: 'Authentication failed.',
  code: 18,
  codeName: 'AuthenticationFailed',
  name: 'MongoError',
  [Symbol(mongoErrorContextSymbol)]: {} }



info [Sun Oct 14 2018 15:33:17 GMT+1100 (AEDT)]  ::[mongoUtils]=> connectToDB() => connecting to db with URL: mongodb://userAdmin:baboon_User_Admin@localhost:27018/king_louie
info [Sun Oct 14 2018 15:33:17 GMT+1100 (AEDT)]  { MongoError: Authentication failed.
    at /Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:581:63
    at authenticateStragglers (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:504:16)
    at Connection.messageHandler (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:540:5)
    at emitMessageHandler (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/connection.js:310:10)
    at Socket.<anonymous> (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/connection.js:453:17)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
    at Socket.Readable.push (_stream_readable.js:208:10)
  ok: 0,
  errmsg: 'Authentication failed.',
  code: 18,
  codeName: 'AuthenticationFailed',
  name: 'MongoError',
  [Symbol(mongoErrorContextSymbol)]: {} }



info [Sun Oct 14 2018 15:33:45 GMT+1100 (AEDT)]  ::[mongoUtils]=> connectToDB() => connecting to db with URL: mongodb://userAdmin:baboon_User_Admin@localhost:27018/king_louie
info [Sun Oct 14 2018 15:33:45 GMT+1100 (AEDT)]  { MongoError: Authentication failed.
    at /Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:581:63
    at authenticateStragglers (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:504:16)
    at Connection.messageHandler (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:540:5)
    at emitMessageHandler (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/connection.js:310:10)
    at Socket.<anonymous> (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/connection.js:453:17)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
    at Socket.Readable.push (_stream_readable.js:208:10)
  ok: 0,
  errmsg: 'Authentication failed.',
  code: 18,
  codeName: 'AuthenticationFailed',
  name: 'MongoError',
  [Symbol(mongoErrorContextSymbol)]: {} }



info [Sun Oct 14 2018 15:38:15 GMT+1100 (AEDT)]  ::[mongoUtils]=> connectToDB() => connecting to db with URL: mongodb://userAdmin:baboon_User_Admin@localhost:27018/king_louie
info [Sun Oct 14 2018 15:38:15 GMT+1100 (AEDT)]  { MongoError: Authentication failed.
    at /Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:581:63
    at authenticateStragglers (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:504:16)
    at Connection.messageHandler (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:540:5)
    at emitMessageHandler (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/connection.js:310:10)
    at Socket.<anonymous> (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/connection.js:453:17)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
    at Socket.Readable.push (_stream_readable.js:208:10)
  ok: 0,
  errmsg: 'Authentication failed.',
  code: 18,
  codeName: 'AuthenticationFailed',
  name: 'MongoError',
  [Symbol(mongoErrorContextSymbol)]: {} }



info [Sun Oct 14 2018 15:38:30 GMT+1100 (AEDT)]  ::[mongoUtils]=> connectToDB() => connecting to db with URL: mongodb://userAdmin:baboon_User_Admin@localhost:27018/king_louie
info [Sun Oct 14 2018 15:38:30 GMT+1100 (AEDT)]  { MongoError: Authentication failed.
    at /Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:581:63
    at authenticateStragglers (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:504:16)
    at Connection.messageHandler (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:540:5)
    at emitMessageHandler (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/connection.js:310:10)
    at Socket.<anonymous> (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/connection.js:453:17)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
    at Socket.Readable.push (_stream_readable.js:208:10)
  ok: 0,
  errmsg: 'Authentication failed.',
  code: 18,
  codeName: 'AuthenticationFailed',
  name: 'MongoError',
  [Symbol(mongoErrorContextSymbol)]: {} }



info [Sun Oct 14 2018 15:38:57 GMT+1100 (AEDT)]  ::[mongoUtils]=> connectToDB() => connecting to db with URL: mongodb://userAdmin:baboon_User_Admin@localhost:27018/king_louie
info [Sun Oct 14 2018 15:38:57 GMT+1100 (AEDT)]  { MongoError: Authentication failed.
    at /Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:581:63
    at authenticateStragglers (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:504:16)
    at Connection.messageHandler (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:540:5)
    at emitMessageHandler (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/connection.js:310:10)
    at Socket.<anonymous> (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/connection.js:453:17)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
    at Socket.Readable.push (_stream_readable.js:208:10)
  ok: 0,
  errmsg: 'Authentication failed.',
  code: 18,
  codeName: 'AuthenticationFailed',
  name: 'MongoError',
  [Symbol(mongoErrorContextSymbol)]: {} }



info [Sun Oct 14 2018 15:39:23 GMT+1100 (AEDT)]  ::[mongoUtils]=> connectToDB() => connecting to db with URL: mongodb://userAdmin:baboon_User_Admin@localhost:27018/king_louie
info [Sun Oct 14 2018 15:39:23 GMT+1100 (AEDT)]  { MongoError: Authentication failed.
    at /Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:581:63
    at authenticateStragglers (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:504:16)
    at Connection.messageHandler (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:540:5)
    at emitMessageHandler (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/connection.js:310:10)
    at Socket.<anonymous> (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/connection.js:453:17)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
    at Socket.Readable.push (_stream_readable.js:208:10)
  ok: 0,
  errmsg: 'Authentication failed.',
  code: 18,
  codeName: 'AuthenticationFailed',
  name: 'MongoError',
  [Symbol(mongoErrorContextSymbol)]: {} }



info [Sun Oct 14 2018 15:40:46 GMT+1100 (AEDT)]  ::[mongoUtils]=> connectToDB() => connecting to db with URL: mongodb://userAdmin:baboon_User_Admin@localhost:27018/king_louie
info [Sun Oct 14 2018 15:40:46 GMT+1100 (AEDT)]  { MongoError: Authentication failed.
    at /Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:581:63
    at authenticateStragglers (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:504:16)
    at Connection.messageHandler (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:540:5)
    at emitMessageHandler (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/connection.js:310:10)
    at Socket.<anonymous> (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/connection.js:453:17)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
    at Socket.Readable.push (_stream_readable.js:208:10)
  ok: 0,
  errmsg: 'Authentication failed.',
  code: 18,
  codeName: 'AuthenticationFailed',
  name: 'MongoError',
  [Symbol(mongoErrorContextSymbol)]: {} }



info [Sun Oct 14 2018 15:41:55 GMT+1100 (AEDT)]  ::[mongoUtils]=> connectToDB() => connecting to db with URL: mongodb://userAdmin:baboon_User_Admin@localhost:27018/king_louie
info [Sun Oct 14 2018 15:41:55 GMT+1100 (AEDT)]  { MongoError: Authentication failed.
    at /Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:581:63
    at authenticateStragglers (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:504:16)
    at Connection.messageHandler (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:540:5)
    at emitMessageHandler (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/connection.js:310:10)
    at Socket.<anonymous> (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/connection.js:453:17)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
    at Socket.Readable.push (_stream_readable.js:208:10)
  ok: 0,
  errmsg: 'Authentication failed.',
  code: 18,
  codeName: 'AuthenticationFailed',
  name: 'MongoError',
  [Symbol(mongoErrorContextSymbol)]: {} }



info [Sun Oct 14 2018 15:43:43 GMT+1100 (AEDT)]  ::[mongoUtils]=> connectToDB() => connecting to db with URL: mongodb://userAdmin:baboon_User_Admin@localhost:27018/king_louie
info [Sun Oct 14 2018 15:43:43 GMT+1100 (AEDT)]  { MongoError: Authentication failed.
    at /Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:581:63
    at authenticateStragglers (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:504:16)
    at Connection.messageHandler (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:540:5)
    at emitMessageHandler (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/connection.js:310:10)
    at Socket.<anonymous> (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/connection.js:453:17)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
    at Socket.Readable.push (_stream_readable.js:208:10)
  ok: 0,
  errmsg: 'Authentication failed.',
  code: 18,
  codeName: 'AuthenticationFailed',
  name: 'MongoError',
  [Symbol(mongoErrorContextSymbol)]: {} }



info [Sun Oct 14 2018 15:44:03 GMT+1100 (AEDT)]  ::[mongoUtils]=> connectToDB() => connecting to db with URL: mongodb://userAdmin:baboon_User_Admin@localhost:27018/king_louie
info [Sun Oct 14 2018 15:44:03 GMT+1100 (AEDT)]  { MongoError: Authentication failed.
    at /Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:581:63
    at authenticateStragglers (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:504:16)
    at Connection.messageHandler (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:540:5)
    at emitMessageHandler (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/connection.js:310:10)
    at Socket.<anonymous> (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/connection.js:453:17)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
    at Socket.Readable.push (_stream_readable.js:208:10)
  ok: 0,
  errmsg: 'Authentication failed.',
  code: 18,
  codeName: 'AuthenticationFailed',
  name: 'MongoError',
  [Symbol(mongoErrorContextSymbol)]: {} }



info [Sun Oct 14 2018 15:44:21 GMT+1100 (AEDT)]  ::[mongoUtils]=> connectToDB() => connecting to db with URL: mongodb://userAdmin:baboon_User_Admin@localhost:27018/king_louie
info [Sun Oct 14 2018 15:44:21 GMT+1100 (AEDT)]  { MongoError: Authentication failed.
    at /Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:581:63
    at authenticateStragglers (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:504:16)
    at Connection.messageHandler (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:540:5)
    at emitMessageHandler (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/connection.js:310:10)
    at Socket.<anonymous> (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/connection.js:453:17)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
    at Socket.Readable.push (_stream_readable.js:208:10)
  ok: 0,
  errmsg: 'Authentication failed.',
  code: 18,
  codeName: 'AuthenticationFailed',
  name: 'MongoError',
  [Symbol(mongoErrorContextSymbol)]: {} }



info [Sun Oct 14 2018 15:44:29 GMT+1100 (AEDT)]  ::[mongoUtils]=> connectToDB() => connecting to db with URL: mongodb://userAdmin:baboon_User_Admin@localhost:27018/king_louie
info [Sun Oct 14 2018 15:44:29 GMT+1100 (AEDT)]  { MongoError: Authentication failed.
    at /Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:581:63
    at authenticateStragglers (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:504:16)
    at Connection.messageHandler (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:540:5)
    at emitMessageHandler (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/connection.js:310:10)
    at Socket.<anonymous> (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/connection.js:453:17)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
    at Socket.Readable.push (_stream_readable.js:208:10)
  ok: 0,
  errmsg: 'Authentication failed.',
  code: 18,
  codeName: 'AuthenticationFailed',
  name: 'MongoError',
  [Symbol(mongoErrorContextSymbol)]: {} }



info [Sun Oct 14 2018 15:44:52 GMT+1100 (AEDT)]  ::[mongoUtils]=> connectToDB() => connecting to db with URL: mongodb://userAdmin:baboon_User_Admin@localhost:27018/king_louie
info [Sun Oct 14 2018 15:44:52 GMT+1100 (AEDT)]  { MongoError: Authentication failed.
    at /Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:581:63
    at authenticateStragglers (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:504:16)
    at Connection.messageHandler (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:540:5)
    at emitMessageHandler (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/connection.js:310:10)
    at Socket.<anonymous> (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/connection.js:453:17)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
    at Socket.Readable.push (_stream_readable.js:208:10)
  ok: 0,
  errmsg: 'Authentication failed.',
  code: 18,
  codeName: 'AuthenticationFailed',
  name: 'MongoError',
  [Symbol(mongoErrorContextSymbol)]: {} }



info [Sun Oct 14 2018 15:44:57 GMT+1100 (AEDT)]  ::[mongoUtils]=> connectToDB() => connecting to db with URL: mongodb://userAdmin:baboon_User_Admin@localhost:27018/king_louie
info [Sun Oct 14 2018 15:44:57 GMT+1100 (AEDT)]  { MongoError: Authentication failed.
    at /Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:581:63
    at authenticateStragglers (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:504:16)
    at Connection.messageHandler (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:540:5)
    at emitMessageHandler (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/connection.js:310:10)
    at Socket.<anonymous> (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/connection.js:453:17)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
    at Socket.Readable.push (_stream_readable.js:208:10)
  ok: 0,
  errmsg: 'Authentication failed.',
  code: 18,
  codeName: 'AuthenticationFailed',
  name: 'MongoError',
  [Symbol(mongoErrorContextSymbol)]: {} }



info [Sun Oct 14 2018 15:45:53 GMT+1100 (AEDT)]  ::[mongoUtils]=> connectToDB() => connecting to db with URL: mongodb://userAdmin:baboon_User_Admin@localhost:27018/king_louie
info [Sun Oct 14 2018 15:45:53 GMT+1100 (AEDT)]  { MongoError: Authentication failed.
    at /Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:581:63
    at authenticateStragglers (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:504:16)
    at Connection.messageHandler (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:540:5)
    at emitMessageHandler (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/connection.js:310:10)
    at Socket.<anonymous> (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/connection.js:453:17)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
    at Socket.Readable.push (_stream_readable.js:208:10)
  ok: 0,
  errmsg: 'Authentication failed.',
  code: 18,
  codeName: 'AuthenticationFailed',
  name: 'MongoError',
  [Symbol(mongoErrorContextSymbol)]: {} }



info [Sun Oct 14 2018 15:46:37 GMT+1100 (AEDT)]  ::[mongoUtils]=> connectToDB() => connecting to db with URL: mongodb://userAdmin:baboon_User_Admin@localhost:27018/king_louie
info [Sun Oct 14 2018 15:46:37 GMT+1100 (AEDT)]  { MongoError: Authentication failed.
    at /Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:581:63
    at authenticateStragglers (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:504:16)
    at Connection.messageHandler (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:540:5)
    at emitMessageHandler (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/connection.js:310:10)
    at Socket.<anonymous> (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/connection.js:453:17)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
    at Socket.Readable.push (_stream_readable.js:208:10)
  ok: 0,
  errmsg: 'Authentication failed.',
  code: 18,
  codeName: 'AuthenticationFailed',
  name: 'MongoError',
  [Symbol(mongoErrorContextSymbol)]: {} }



info [Sun Oct 14 2018 15:47:11 GMT+1100 (AEDT)]  ::[mongoUtils]=> connectToDB() => connecting to db with URL: mongodb://userAdmin:baboon_User_Admin@localhost:27018/king_louie
info [Sun Oct 14 2018 15:47:11 GMT+1100 (AEDT)]  { MongoError: Authentication failed.
    at /Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:581:63
    at authenticateStragglers (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:504:16)
    at Connection.messageHandler (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:540:5)
    at emitMessageHandler (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/connection.js:310:10)
    at Socket.<anonymous> (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/connection.js:453:17)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
    at Socket.Readable.push (_stream_readable.js:208:10)
  ok: 0,
  errmsg: 'Authentication failed.',
  code: 18,
  codeName: 'AuthenticationFailed',
  name: 'MongoError',
  [Symbol(mongoErrorContextSymbol)]: {} }



info [Sun Oct 14 2018 15:52:09 GMT+1100 (AEDT)]  ::[mongoUtils]=> connectToDB() => connecting to db with URL: mongodb://userAdmin:baboon_User_Admin@localhost:27018/king_louie
info [Sun Oct 14 2018 15:52:10 GMT+1100 (AEDT)]  { MongoError: Authentication failed.
    at /Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:581:63
    at authenticateStragglers (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:504:16)
    at Connection.messageHandler (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:540:5)
    at emitMessageHandler (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/connection.js:310:10)
    at Socket.<anonymous> (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/connection.js:453:17)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
    at Socket.Readable.push (_stream_readable.js:208:10)
  ok: 0,
  errmsg: 'Authentication failed.',
  code: 18,
  codeName: 'AuthenticationFailed',
  name: 'MongoError',
  [Symbol(mongoErrorContextSymbol)]: {} }



info [Sun Oct 14 2018 15:52:27 GMT+1100 (AEDT)]  ::[mongoUtils]=> connectToDB() => connecting to db with URL: mongodb://userAdmin:baboon_User_Admin@localhost:27018/king_louie
info [Sun Oct 14 2018 15:52:27 GMT+1100 (AEDT)]  { MongoError: Authentication failed.
    at /Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:581:63
    at authenticateStragglers (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:504:16)
    at Connection.messageHandler (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:540:5)
    at emitMessageHandler (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/connection.js:310:10)
    at Socket.<anonymous> (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/connection.js:453:17)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
    at Socket.Readable.push (_stream_readable.js:208:10)
  ok: 0,
  errmsg: 'Authentication failed.',
  code: 18,
  codeName: 'AuthenticationFailed',
  name: 'MongoError',
  [Symbol(mongoErrorContextSymbol)]: {} }



info [Sun Oct 14 2018 15:53:14 GMT+1100 (AEDT)]  ::[mongoUtils]=> connectToDB() => connecting to db with URL: mongodb://userAdmin:baboon_User_Admin@localhost:27018/king_louie
info [Sun Oct 14 2018 15:53:14 GMT+1100 (AEDT)]  { MongoError: Authentication failed.
    at /Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:581:63
    at authenticateStragglers (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:504:16)
    at Connection.messageHandler (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:540:5)
    at emitMessageHandler (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/connection.js:310:10)
    at Socket.<anonymous> (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/connection.js:453:17)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
    at Socket.Readable.push (_stream_readable.js:208:10)
  ok: 0,
  errmsg: 'Authentication failed.',
  code: 18,
  codeName: 'AuthenticationFailed',
  name: 'MongoError',
  [Symbol(mongoErrorContextSymbol)]: {} }



info [Sun Oct 14 2018 15:53:32 GMT+1100 (AEDT)]  ::[mongoUtils]=> connectToDB() => connecting to db with URL: mongodb://userAdmin:baboon_User_Admin@localhost:27018/king_louie
info [Sun Oct 14 2018 15:53:32 GMT+1100 (AEDT)]  { MongoError: Authentication failed.
    at /Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:581:63
    at authenticateStragglers (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:504:16)
    at Connection.messageHandler (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:540:5)
    at emitMessageHandler (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/connection.js:310:10)
    at Socket.<anonymous> (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/connection.js:453:17)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
    at Socket.Readable.push (_stream_readable.js:208:10)
  ok: 0,
  errmsg: 'Authentication failed.',
  code: 18,
  codeName: 'AuthenticationFailed',
  name: 'MongoError',
  [Symbol(mongoErrorContextSymbol)]: {} }



info [Sun Oct 14 2018 15:56:08 GMT+1100 (AEDT)]  ::[mongoUtils]=> connectToDB() => connecting to db with URL: mongodb://userAdmin:baboon_User_Admin@localhost:27018/king_louie
info [Sun Oct 14 2018 15:56:08 GMT+1100 (AEDT)]  { MongoError: Authentication failed.
    at /Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:581:63
    at authenticateStragglers (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:504:16)
    at Connection.messageHandler (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:540:5)
    at emitMessageHandler (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/connection.js:310:10)
    at Socket.<anonymous> (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/connection.js:453:17)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
    at Socket.Readable.push (_stream_readable.js:208:10)
  ok: 0,
  errmsg: 'Authentication failed.',
  code: 18,
  codeName: 'AuthenticationFailed',
  name: 'MongoError',
  [Symbol(mongoErrorContextSymbol)]: {} }



info [Sun Oct 14 2018 15:57:38 GMT+1100 (AEDT)]  ::[mongoUtils]=> connectToDB() => connecting to db with URL: mongodb://userAdmin:baboon_User_Admin@localhost:27018/king_louie
info [Sun Oct 14 2018 15:57:38 GMT+1100 (AEDT)]  { MongoError: Authentication failed.
    at /Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:581:63
    at authenticateStragglers (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:504:16)
    at Connection.messageHandler (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:540:5)
    at emitMessageHandler (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/connection.js:310:10)
    at Socket.<anonymous> (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/connection.js:453:17)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
    at Socket.Readable.push (_stream_readable.js:208:10)
  ok: 0,
  errmsg: 'Authentication failed.',
  code: 18,
  codeName: 'AuthenticationFailed',
  name: 'MongoError',
  [Symbol(mongoErrorContextSymbol)]: {} }



info [Sun Oct 14 2018 16:05:23 GMT+1100 (AEDT)]  ::[mongoUtils]=> connectToDB() => connecting to db with URL: mongodb://userAdmin:baboon_User_Admin@localhost:27018/king_louie
info [Sun Oct 14 2018 16:05:23 GMT+1100 (AEDT)]  { MongoError: Authentication failed.
    at /Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:581:63
    at authenticateStragglers (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:504:16)
    at Connection.messageHandler (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:540:5)
    at emitMessageHandler (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/connection.js:310:10)
    at Socket.<anonymous> (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/connection.js:453:17)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
    at Socket.Readable.push (_stream_readable.js:208:10)
  ok: 0,
  errmsg: 'Authentication failed.',
  code: 18,
  codeName: 'AuthenticationFailed',
  name: 'MongoError',
  [Symbol(mongoErrorContextSymbol)]: {} }



info [Sun Oct 14 2018 16:05:57 GMT+1100 (AEDT)]  ::[mongoUtils]=> connectToDB() => connecting to db with URL: mongodb://userAdmin:baboon_User_Admin@localhost:27018/king_louie
info [Sun Oct 14 2018 16:05:57 GMT+1100 (AEDT)]  { MongoError: Authentication failed.
    at /Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:581:63
    at authenticateStragglers (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:504:16)
    at Connection.messageHandler (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:540:5)
    at emitMessageHandler (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/connection.js:310:10)
    at Socket.<anonymous> (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/connection.js:453:17)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
    at Socket.Readable.push (_stream_readable.js:208:10)
  ok: 0,
  errmsg: 'Authentication failed.',
  code: 18,
  codeName: 'AuthenticationFailed',
  name: 'MongoError',
  [Symbol(mongoErrorContextSymbol)]: {} }


info [Sun Oct 14 2018 16:07:21 GMT+1100 (AEDT)]  ::[mongoUtils]=> connectToDB() => connecting to db with URL: mongodb://userAdmin:baboon_User_Admin@localhost:27018/king_louie
info [Sun Oct 14 2018 16:07:21 GMT+1100 (AEDT)]  { MongoError: Authentication failed.
    at /Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:581:63
    at authenticateStragglers (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:504:16)
    at Connection.messageHandler (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:540:5)
    at emitMessageHandler (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/connection.js:310:10)
    at Socket.<anonymous> (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/connection.js:453:17)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
    at Socket.Readable.push (_stream_readable.js:208:10)
  ok: 0,
  errmsg: 'Authentication failed.',
  code: 18,
  codeName: 'AuthenticationFailed',
  name: 'MongoError',
  [Symbol(mongoErrorContextSymbol)]: {} }



info [Sun Oct 14 2018 16:09:29 GMT+1100 (AEDT)]  ::[remoteService]=> #--- Server Listening on http://localhost port: 8001
info [Sun Oct 14 2018 16:09:29 GMT+1100 (AEDT)]  ::[mongoUtils]=> connectToDB() => connecting to db with URL: mongodb://userAdmin:baboon_User_Admin@localhost:27018/king_louie
info [Sun Oct 14 2018 16:09:29 GMT+1100 (AEDT)]  { MongoError: Authentication failed.
    at /Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:581:63
    at authenticateStragglers (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:504:16)
    at Connection.messageHandler (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:540:5)
    at emitMessageHandler (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/connection.js:310:10)
    at Socket.<anonymous> (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/connection.js:453:17)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
    at Socket.Readable.push (_stream_readable.js:208:10)
  ok: 0,
  errmsg: 'Authentication failed.',
  code: 18,
  codeName: 'AuthenticationFailed',
  name: 'MongoError',
  [Symbol(mongoErrorContextSymbol)]: {} }
info [Sun Oct 14 2018 16:09:29 GMT+1100 (AEDT)]  ::[remoteService]=> connectToDB()=>  Connected to king_louie



info [Sun Oct 14 2018 16:10:11 GMT+1100 (AEDT)]  ::[remoteService]=> #--- Server Listening on http://localhost port: 8001
info [Sun Oct 14 2018 16:10:11 GMT+1100 (AEDT)]  ::[mongoUtils]=> connectToDB() => connecting to db with URL: mongodb://userAdmin:baboon_User_Admin@localhost:27018/king_louie
info [Sun Oct 14 2018 16:10:11 GMT+1100 (AEDT)]  { MongoError: Authentication failed.
    at /Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:581:63
    at authenticateStragglers (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:504:16)
    at Connection.messageHandler (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:540:5)
    at emitMessageHandler (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/connection.js:310:10)
    at Socket.<anonymous> (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/connection.js:453:17)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
    at Socket.Readable.push (_stream_readable.js:208:10)
  ok: 0,
  errmsg: 'Authentication failed.',
  code: 18,
  codeName: 'AuthenticationFailed',
  name: 'MongoError',
  [Symbol(mongoErrorContextSymbol)]: {} }
info [Sun Oct 14 2018 16:10:11 GMT+1100 (AEDT)]  ::[remoteService]=> connectToDB()=>  Connected to king_louie



info [Sun Oct 14 2018 16:15:31 GMT+1100 (AEDT)]  ::[remoteService]=> #--- Server Listening on http://localhost port: 8001
info [Sun Oct 14 2018 16:15:31 GMT+1100 (AEDT)]  ::[mongoUtils]=> connectToDB() => connecting to db with URL: mongodb://userAdmin:baboon_User_Admin@localhost:27018/king_louie
info [Sun Oct 14 2018 16:15:31 GMT+1100 (AEDT)]  { MongoError: Authentication failed.
    at /Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:581:63
    at authenticateStragglers (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:504:16)
    at Connection.messageHandler (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:540:5)
    at emitMessageHandler (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/connection.js:310:10)
    at Socket.<anonymous> (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/connection.js:453:17)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
    at Socket.Readable.push (_stream_readable.js:208:10)
  ok: 0,
  errmsg: 'Authentication failed.',
  code: 18,
  codeName: 'AuthenticationFailed',
  name: 'MongoError',
  [Symbol(mongoErrorContextSymbol)]: {} }
info [Sun Oct 14 2018 16:15:31 GMT+1100 (AEDT)]  ::[remoteService]=> connectToDB()=>  Connected to king_louie



info [Sun Oct 14 2018 16:15:43 GMT+1100 (AEDT)]  ::[remoteService]=> #--- Server Listening on http://localhost port: 8001
info [Sun Oct 14 2018 16:15:43 GMT+1100 (AEDT)]  ::[mongoUtils]=> connectToDB() => connecting to db with URL: mongodb://userAdmin:baboon_User_Admin@localhost:27018/king_louie
info [Sun Oct 14 2018 16:15:43 GMT+1100 (AEDT)]  { MongoError: Authentication failed.
    at /Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:581:63
    at authenticateStragglers (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:504:16)
    at Connection.messageHandler (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:540:5)
    at emitMessageHandler (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/connection.js:310:10)
    at Socket.<anonymous> (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/connection.js:453:17)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
    at Socket.Readable.push (_stream_readable.js:208:10)
  ok: 0,
  errmsg: 'Authentication failed.',
  code: 18,
  codeName: 'AuthenticationFailed',
  name: 'MongoError',
  [Symbol(mongoErrorContextSymbol)]: {} }
info [Sun Oct 14 2018 16:15:43 GMT+1100 (AEDT)]  ::[remoteService]=> connectToDB()=>  Connected to king_louie



info [Sun Oct 14 2018 16:19:51 GMT+1100 (AEDT)]  ::[remoteService]=> #--- Server Listening on http://localhost port: 8001
info [Sun Oct 14 2018 16:19:51 GMT+1100 (AEDT)]  ::[mongoUtils]=> connectToDB() => connecting to db with URL: mongodb://userAdmin:baboon_User_Admin@localhost:27018/king_louie
info [Sun Oct 14 2018 16:19:51 GMT+1100 (AEDT)]  { MongoError: Authentication failed.
    at /Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:581:63
    at authenticateStragglers (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:504:16)
    at Connection.messageHandler (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:540:5)
    at emitMessageHandler (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/connection.js:310:10)
    at Socket.<anonymous> (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/connection.js:453:17)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
    at Socket.Readable.push (_stream_readable.js:208:10)
  ok: 0,
  errmsg: 'Authentication failed.',
  code: 18,
  codeName: 'AuthenticationFailed',
  name: 'MongoError',
  [Symbol(mongoErrorContextSymbol)]: {} }
info [Sun Oct 14 2018 16:19:51 GMT+1100 (AEDT)]  ::[remoteService]=> connectToDB()=>  Connected to king_louie



info [Sun Oct 14 2018 16:23:16 GMT+1100 (AEDT)]  ::[remoteService]=> #--- Server Listening on http://localhost port: 8001
info [Sun Oct 14 2018 16:23:16 GMT+1100 (AEDT)]  ::[mongoUtils]=> connectToDB() => connecting to db with URL: mongodb://userAdmin:baboon_User_Admin@localhost:27018/king_louie
info [Sun Oct 14 2018 16:23:16 GMT+1100 (AEDT)]  { MongoError: Authentication failed.
    at /Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:581:63
    at authenticateStragglers (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:504:16)
    at Connection.messageHandler (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/pool.js:540:5)
    at emitMessageHandler (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/connection.js:310:10)
    at Socket.<anonymous> (/Users/claudio/baboon_remoteServer/node_modules/mongodb-core/lib/connection/connection.js:453:17)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
    at Socket.Readable.push (_stream_readable.js:208:10)
  ok: 0,
  errmsg: 'Authentication failed.',
  code: 18,
  codeName: 'AuthenticationFailed',
  name: 'MongoError',
  [Symbol(mongoErrorContextSymbol)]: {} }
info [Sun Oct 14 2018 16:23:16 GMT+1100 (AEDT)]  ::[remoteService]=> connectToDB()=>  Connected to king_louie
